src/System.Runtime.Extensions/src/Resources/Strings.resx

```diff
@@ -187,7 +190,7 @@
     <value>'{0}' must be greater than zero.</value>
   </data>
   <data name="IO_PathTooLong" xml:space="preserve">
-    <value>The specified path, file name, or both are too long. The fully qualified file name must be less than 260 characters, and the directory name must be less than 248 characters.</value>
+    <value>The specified path, file name, or both are too long. The fully-qualified file name must be less than {0} characters, and the directory name must be less than {1} characters.</value>
```

Comment:
I think there are some hits on this that didn't get edited:
* src/Common/src/Interop/Unix/Interop.IOErrors.cs
* src/Common/src/System/IO/Win32Marshal.cs
* src/System.IO.FileSystem/src/System/IO/WinRTFileSystem.cs
* src/System.IO.FileSystem/src/System/IO/Win32FileSystem.cs

Not to mention that there are a half-dozen or so copies of this string across the various libraries.